_component_design/footer.md

```diff
@@ -4,83 +4,53 @@ parent: Navigation
-global functionality on all of our platforms.
-
-#### Best practices for footers
+### Best practices for footers
 
-Place the footer at the bottom of every page. If the page uses left-hand
-navigation, ensure the footer is to the right of the screen, and spans the
-length of the screen. If using top navigation, the footer container must span
-the bottom of the entire page. Ensure the footer contains the following:
+* Footer items often get overlooked in customer-facing contexts, so avoid placing critical or dynamic content there.
+* For customer-facing applications the footer contains the following:
 
-* Rackspace copyright and year
-* A link to website terms and conditions
-* A link to the privacy policy
+  * Rackspace copyright and year
+  * A link to [Website Terms](https://www.rackspace.com/information/legal/websiteterms) and conditions
+  * A link to the [Privacy Policy](https://www.rackspace.com/information/legal/privacystatement)
 
 {% endcolumn %}
 
 {% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
 
-{% figure [caption:"Thumbnail."] [class:"image bg-light border"] %}
-  ![]({{site.url}}/assets/images/components/content-areas/footer/footer-thumbnail.svg){:width="80%"}
-{% endfigure %}
-
-{% endcolumn %}
-</div>
-
-### Specifications
-<div class="hxRow">
-{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
-
-#### Footer scope
-
-The footer must contain copyright information with links to Rackspace
-website terms and privacy policy.
-
-* <a href="https://www.rackspace.com/information/legal/websiteterms" target="_blank">Website Terms</a> 
-* <a href="https://www.rackspace.com/information/legal/privacystatement" target="_blank">Privacy Policy</a>
-
-For customer applications, do not add additional functionality to the footer.
-{% endcolumn %}
-
-{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
-{% figure [caption:"The footer rests on the bottom of the page it is on."] [class:"image bg-light border"] %}
+{% figure [caption:"Place the footer at the bottom of every page."] [class:"image bg-light border"] %}
   ![]({{site.url}}/assets/images/components/content-areas/footer/footer-scope.svg){:width="80%"}
 {% endfigure %}
+
 {% endcolumn %}
 </div>
 
-### Footer specifications
+## Footer specifications
 <div class="hxRow">
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-All footer content starts after the gutter at 20px. The footer container is
-40px tall with text centered vertically. Items are spaced 32px apart.
-Links are colored cyan 500 on dark backgrounds and behave as described in
-the Links documentation.
+* The footer must span the width of the page
+* All footer content starts after the gutter at 20px
+* The footer container is 40px tall with text centered vertically
+* Footer items are spaced 32px apart
+* Links are colored cyan 500 on dark backgrounds and behave as described in the Links documentation.
```

Comment:
As noted on another PR review tonight, I'd put the color in code as that's how I think of it here (as in, the coding is what's being referenced). I could see the argument either way, though.
```markdown
are colored `cyan 500` on dark backgrounds
```

Reply:
this is a good suggestion, done.